pytorch_object_detection/detection_models/configs.py

```diff
@@ -0,0 +1,7 @@
+from torchvision.models.detection import FasterRCNN_ResNet50_FPN_V2_Weights,\
+    RetinaNet_ResNet50_FPN_Weights
+
+
+class DetectionModelsWeights:
```

Comment:
Maybe use Enum?

Reply:
When use Enum I can not get meta of classess of weights, so I removed it

Reply:
Do Enum, if any issues - log here

Reply:
done